rgbd/src/serialization.cpp

```diff
@@ -278,6 +276,15 @@ bool deserialize(tue::serialization::InputArchive& a, Image& image)
         return false;
     }
 
+    if (cam_type == CAMERA_MODEL_PINHOLE)
+    {
+        // now that the image size is known add it to the camera model
```

Comment:
This suggest that somewhere earlier in the code there is a comment that we don't know the resolution yet. That comment is missing.